packages/@adobe/spectrum-css-temp/components/inlinealert/index.css

```diff
@@ -0,0 +1,117 @@
+
+  /* Spacing*/
+  --spectrum-inlinealert-spacing-edge-to-text: var(--spectrum-global-dimension-static-size-300);
+  --spectrum-inlinealert-spacing-header-to-icon: var(--spectrum-global-dimension-static-size-300);
+  --spectrum-inlinealert-spacing-header-content-button: var(--spectrum-global-dimension-static-size-200);
+
+  /* Color */
+  --spectrum-inlinealert-background-color: var(--spectrum-alias-background-color-default);
+  --spectrum-inlinealert-border-and-icon-color: var(--spectrum-gray-visual-color);
+  --spectrum-inlinealert-header-color: var(--spectrum-alias-heading-text-color);
+  --spectrum-inlinealert-content-color: var(--spectrum-alias-text-color);
+  --spectrum-inlinealert-border-and-icon-color-info: var(--spectrum-informative-visual-color);
+  --spectrum-inlinealert-border-and-icon-color-positive: var(--spectrum-positive-visual-color);
+  --spectrum-inlinealert-border-and-icon-color-notice: var(--spectrum-notice-visual-color);
+  --spectrum-inlinealert-border-and-icon-color-negative: var(--spectrum-negative-visual-color);
+}
+
+.spectrum-InLineAlert {
+  position: relative;
+
+  display: inline-block;
+  box-sizing: border-box;
+  min-inline-size: var(--spectrum-inlinealert-min-inline-size);
+
+  padding-block: var(--spectrum-inlinealert-spacing-edge-to-text);
+  padding-inline: var(--spectrum-inlinealert-spacing-edge-to-text);
+
+  border-block-width: var(--spectrum-inlinealert-border-width);
+  border-inline-width: var(--spectrum-inlinealert-border-width);
+  border-style: solid;
+  border-radius: var(--spectrum-inlinealert-border-radius);
+}
+
+.spectrum-InLineAlert .spectrum-InLineAlert-grid {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto auto;
+  column-gap: var(--spectrum-inlinealert-spacing-header-to-icon);
+  grid-template-areas:
+    "header      icon       "
+    "content     content    ";
+  width: 100%;
+}
+
+@media (forced-colors: active) {
+  .spectrum-InLineAlert {
+    --highcontrast-inlinealert-background-color: Background;
+    --highcontrast-inlinealert-header-color: CanvasText;
+    --highcontrast-inlinealert-content-color: CanvasText;
+    --highcontrast-inlinealert-border-and-icon-color: ButtonBorder;
+  }
+}
+
+.spectrum-InLineAlert-icon {
+  inline-size: var(--spectrum-inlinealert-icon-size);
+  block-size: var(--spectrum-inlinealert-icon-size);
+}
+
+.spectrum-InLineAlert-header {
+  display: block;
+
+  font-weight: var(--spectrum-inlinealert-heading-font-weight);
+  font-style: var(--spectrum-inlinealert-heading-font-style);
+  font-size: var(--spectrum-inlinealert-heading-font-size);
+  line-height: var(--spectrum-inlinealert-heading-line-height);
+  text-transform: none;
+
+  min-block-size: var(--spectrum-inlinealert-header-min-block-size);
+}
+
+.spectrum-InLineAlert-content {
+  display: block;
+  margin-block-start: var(--spectrum-inlinealert-spacing-header-content-button);
```

Comment:
That is the var name `spectrum-css` used since it still supports buttons in the footer. Since RSP implmentation does not allow buttons I can re-name this var to be a better match.

Reply:
if that's the only thing in spectrum-css that gives the correct margin there, maybe we keep it then... just to make future reconciling marginally easier :-/